core/src/sumeragi/mod.rs

```diff
@@ -61,7 +61,7 @@ trait Consensus {
 /// `Sumeragi` is the implementation of the consensus.
 #[derive(Debug)]
 pub struct Sumeragi {
-    internal: SumeragiWithFault<NoFault>,
+    pub internal: SumeragiWithFault<NoFault>,
```

Comment:
@Arjentix We should eventually get rid of these. I personally wouldn't mind getting rid of it after merging this.

Reply:
@appetrosyan, I'm not in the context. Do you mean `FaultInjectionTrait` or what?

Reply:
Yes. Specifically if we want to do fault injection, we should do it via a separate binary, not the way we do it now.